### DISCUSSION TITLE
Restricting access to the main page

### DISCUSSION THREAD

**Discussion by predlogio** [requarks/wiki#3790] [Help / Questions]
Good afternoon! I'm starting to learn the Wiki.js and use it to document collaborative projects with colleagues. I studied all the documentation, but I could not find how to close the home page from everyone. At the moment, when switching to my wiki.domain.com the first wiki page opens immediately. But to start working, I need to grant access to the wiki only after authorization, please tell me how this can be done?

OS: Debian 10 and Docker
Wiki.js version: 2.5.197
Database engine: PostgreSQL 11.11

**Comment by PaulD987** [comment]
On the wiki I look after we have removed all permissions from the Guests group, which means that anyone who isn't authenticated can't access anything on the wiki. Is that was you require?

**Reply by predlogio** [reply]
Yes, that's what I need.  I removed all permissions from the guest group, but the home page still opens.  Can you post a link to this topic?

**Reply by PaulD987** [reply]
I presume you clicked the "update group" after removing permissions from the group? I also deleted all assigned roles on the "Page Rules" tab, but don't think that is needed.
![image](https://user-images.githubusercontent.com/73899671/113315120-d9504200-9304-11eb-9c2c-a37ac4584417.png)
